Library/Homebrew/brew.sh

```diff
@@ -159,24 +182,44 @@ else
 
   # Ensure the system Curl is a version that supports modern HTTPS certificates.
   HOMEBREW_MINIMUM_CURL_VERSION="7.41.0"
-  system_curl_version_output="$($(command -v curl) --version 2>/dev/null)"
-  system_curl_name_and_version="${system_curl_version_output%% (*}"
-  if [[ $(numeric "${system_curl_name_and_version##* }") -lt $(numeric "$HOMEBREW_MINIMUM_CURL_VERSION") ]]
+  curl_version_output="$($HOMEBREW_CURL --version 2>/dev/null)"
+  curl_name_and_version="${curl_version_output%% (*}"
+  if [[ $(numeric "${curl_name_and_version##* }") -lt $(numeric "$HOMEBREW_MINIMUM_CURL_VERSION") ]]
   then
-    HOMEBREW_SYSTEM_CURL_TOO_OLD="1"
-    HOMEBREW_FORCE_BREWED_CURL="1"
+    if [[ -z $HOMEBREW_CURL_PATH ]]; then
```

Comment:
Thoughts on also handling `HOMEBREW_FORCE_BREWED_CURL` being too old here, too? Or shall we assume that'll never happen.

Reply:
`HOMEBREW_FORCE_BREWED_CURL` is a flag that instructs Homebrew to use `"$HOMEBREW_PREFIX/opt/curl/bin/curl"`. It should be fine, I think

Reply:
Yeh, I guess I was just thinking "what if that's too old" but that seems unlikely enough to ignore (if you agree).

Reply:
Yes, I think we can ignore this possibility.